domain/modeldefaults/state/state.go

```diff
@@ -48,14 +59,9 @@ func (s *State) ModelCloudDefaults(
 	}
 
 	cloudDefaultsStmt := `
-SELECT cloud_defaults.key,
-       cloud_defaults.value
+SELECT cloud_defaults.key, cloud_defaults.value
 FROM cloud_defaults
-INNER JOIN cloud
-ON cloud.uuid = cloud_defaults.cloud_uuid
-INNER JOIN model m
-ON m.cloud_uuid = cloud.uuid
-WHERE m.uuid = ?
+WHERE cloud_defaults.cloud_uuid = ?
 `
 
 	err = db.StdTxn(ctx, func(ctx context.Context, tx *sql.Tx) error {
```

Comment:
These are some of the last remaining usages of StdTxn, I'm happy to put up a PR after this one is in to replace them with SQLair.

Reply:
Yeah, I thought about fixing but the PR was already big enough.